libraries/SITL/SIM_Precland.h

```diff
@@ -37,6 +37,7 @@ class SIM_Precland {
     uint32_t last_update_ms() const { return _last_update_ms; }
 
     const Vector3d &get_target_position() const { return _target_pos; }
+    float get_target_distance() const;
```

Comment:
I know we're short on comments in here, but we should add one here.

```suggestion
    // distance to target (in metres).  See DST_EN parameter
    float get_target_distance() const;
```